_maps/map_files/IceBoxStation/IceBoxStation.dmm

```diff
@@ -50112,8 +49469,12 @@
 /turf/closed/wall/r_wall,
 /area/science/mixing/chamber)
 "fep" = (
-/obj/structure/window/reinforced,
-/turf/open/floor/engine,
+/obj/machinery/camera{
+	c_tag = "Xenobiology Pens Observastion - Starboard Aft";
```

Comment:
Observation, same for all other instances of "Observa_s_tion"

Reply:
fixed up fixed up the wording